src/components/ComposedModal/ComposedModal.jsx

```diff
@@ -228,14 +162,18 @@ class ComposedModal extends React.Component {
     return isFetchingData ? (
       <Loading />
     ) : (
-      <StyledModal
+      <CarbonComposedModal
         {...props}
         open={open}
         onClose={this.doNotClose}
-        className={classnames(className, {
-          'error-modal': type === 'warn',
-          'big-modal': isLarge,
-        })}
+        className={classnames(
+          className,
+          {
+            'error-modal': type === 'warn',
+            'big-modal': isLarge,
```

Comment:
I would swap these out for BEM styled modifiers to the main class name. 
(eg. `${iotPrefix}--composed-modal__large` )

Reply:
[some additional info on BEM from the carbon developer handbook](https://github.com/carbon-design-system/carbon/blob/e7eb4754c9434c28a1da447c73d8b6d946a2569f/docs/developer-handbook.md#coding-style) - see the `SCSS` section